packages/canvas/package.json

```
@@ -50,6 +50,7 @@
     "deep-parse-json": "1.0.1",
     "faunadb": "4.0.0",
     "graphql-tag": "2.11.0",
+    "rollup-plugin-node-polyfills": "^0.2.1",
```

```suggestion
    "rollup-plugin-node-polyfills": "0.2.1",
```

Need fixed versions on all package jsons. Cant remember exactly why but i think it had something to do with lock file always having a diff if we didn't